test/test_transforms_tensor.py

```diff
@@ -309,11 +311,10 @@ def test_x_crop(fn, method, out_length, size, device):
 
 
 @pytest.mark.parametrize('method', ["FiveCrop", "TenCrop"])
-def test_x_crop_save(method):
+def test_x_crop_save(method, tmpdir):
```

Comment:
Just to clarify: for these functions that directly use the `tmpdir` fixture and do not indirectly call `_test_class_op` of `_test_op`, it's nice to have `tmpdir` and not to rely on `get_tmp_dir`, so no need to revert the changes for these :)

Reply:
Yes, per above comment it is edited.